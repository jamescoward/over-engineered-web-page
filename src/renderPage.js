const renderPage = () => {
  return `
    <html>
      <head>
        <title>Overengineered Webpage</title>
      </head>
      <body>
        <h1>Overengineered webpage</h1>
      </body>
    </html>
  `
};

module.exports = {
  renderPage,
};
