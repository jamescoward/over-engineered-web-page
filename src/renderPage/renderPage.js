const renderPage = () => {
  return `
    <html>
      <head>
        <title>Overengineered Webpage</title>
      </head>
      <body>
        <h1>Overengineered webpage - Shiny Version</h1>
      </body>
    </html>
  `;
};

module.exports = {
  renderPage,
};
