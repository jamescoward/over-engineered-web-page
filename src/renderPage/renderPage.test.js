import { renderPage } from "./renderPage";

it("renders correctly", () => {
  const renderedOutput = renderPage();
  expect(renderedOutput).toMatchSnapshot();
});
