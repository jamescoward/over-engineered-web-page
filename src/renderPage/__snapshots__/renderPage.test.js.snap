// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
"
    <html>
      <head>
        <title>Overengineered Webpage</title>
      </head>
      <body>
        <h1>Overengineered webpage</h1>
      </body>
    </html>
  "
`;
